content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,834 @@
+
+Let's add the motion vector shader and then edit its vertex shader:
+
+1. Open `VegetationBending.materialtype`. At the bottom of the `shaders` list, add the motion vector pass:
+   
+   ```json
+   {
+      "file": "./MeshMotionVectorVegetationBending.shader",
+      "tag": "MeshMotionVector"
+   }
+   ```
+
+1. Open `MeshMotionVectorVegetationBending.azsl`.
+
+1. Add the call to `ProcessBending`, just as we did in the previous steps with the other shaders. Under the declaration for `float4 prevWorldPosition`, but above `OUT.m_worldPos`, add:
+
+   ```hlsl
+   float currentTime = SceneSrg::m_time;
+   worldPosition = ProcessBending(currentTime, IN.m_position, IN.m_normal, IN.m_optional_color, worldPosition, objectToWorld);
+   ```
+
+1. For our motion vectors to work we need to find the previous world position. We want to perform bending on the vertex at the `IN.m_position` but at the previous frame time, with the previous world position. We can use `SceneSrg::m_prevTime` to get the previous frame time. Call our `ProcessBending` function again, but with the appropriate time and world position:
+
+   ```hlsl
+   float currentTime = SceneSrg::m_time;
+   worldPosition = ProcessBending(currentTime, IN.m_position, IN.m_normal, IN.m_optional_color, worldPosition, objectToWorld);
+   float prevTime = SceneSrg::m_prevTime;
+   prevWorldPosition = ProcessBending(prevTime, IN.m_position, IN.m_normal, IN.m_optional_color, prevWorldPosition, prevObjectToWorld);
+   ```
+
+1. Take a look at the pixel shader to see how the motion vector is calculated! There is no need to edit the pixel shader.
+
+Amazing, we have added everything we need to add for motion vectors! However, if you open the **Editor** and just view the tree, you'll see that there is no difference. We can, however, observe that the motion vector pass works by using the **pass tree visualizer**.
+1. Open the Editor and press **Ctrl-G** to enter gameplay mode.
+1. Press the **Home** key on your keyboard. This brings up the toolbar at the top.
+1. Select **Atom Tools** > **Pass Viewer**.
+1. In the pop-up **PassTree View**, enable **Preview Attachment** and **Show Pass Attachments**.
+1. In the **PassTree**, find *MotionVectorPass* > *MeshMotionVectorPass* and select the line with `CameraMotion`. 
+1. Ensure you are viewing your tree. Note that you probably won't see much except black on the bottom left preview, because the motion vectors are very small since our tree only moves minimally. However, if you move the camera around or translate the tree quickly, you may see some motion vectors pop up. You can also open `MeshMotionVectorVegetationBending.azsl` and scale `OUT.m_motion` in the pixel shader to ensure that the motion vectors' directions are working properly.
```

Comment:
For long steps like this, consider breaking it up by adding a new line in between the shorter paragraphs. You can indent the paragraphs so they are aligned with the step. The space helps with readability. 

Some of this information may be better in a note callout box as well.